fastparquet/benchmarks/columns.py

```diff
@@ -19,11 +23,11 @@ def measure(name, result):
 
 
 def time_column():
-    with tmpdir() as tempdir:
+    with Tmpdir() as tempdir:
         result = {}
-        fn = join_path(tempdir, 'temp.parq')
+        fn = join_path(tempdir.name, 'temp.parq')
         n = 10000000
-        r = np.random.randint(-1e10, 1e10, n)
+        r = np.random.randint(-1e10, 1e10, n, dtype='int64')
```

Comment:
win32 will default to `int32`, now it is explicit.  I do not think this changes the meaning on 64bit